src/components/WrapEtherWidget/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useMemo } from 'react'
+import { WraptorComponent } from '@w3stside/wraptor'
+import { useWalletConnection } from 'hooks/useWalletConnection'
+
+import { tokenListApi } from 'api'
+import { Network } from 'types'
+
+const WrapEther: React.FC = () => {
+  const { userAddress, networkId, web3 } = useWalletConnection()
+
+  const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
+  const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId), [fallBackNetworkId])
+
+  const contractAddress = useMemo(() => tokens.find(token => token.symbol === 'WETH')?.address, [tokens])
```

Comment:
As we only use `tokens` here, put it all into one `useMemo`

Reply:
ok!